main.go

```diff
@@ -86,6 +87,7 @@ func main() {
 	flag.StringVar(&config.Scheme, "scheme", "http", "The URL scheme on which receive components accept write requests")
 	flag.StringVar(&config.InternalAddr, "internal-addr", ":8080", "The address on which internal server runs")
 	flag.BoolVar(&config.AllowOnlyReadyReplicas, "allow-only-ready-replicas", false, "Populate only Ready receiver replicas in the hashring configuration")
+	flag.BoolVar(&config.AllowDynamicScaling, "allow-dynamic-scaling", false, "Update the hashring configuration on scale down events.")
```

Comment:
I guess not only on scale down event, but on all disruptions? I think this is related to my previous suggestion. Ideally this should be then named `allow-scaling-on-disruption` but it brings me again to whether all of these scenarios are disruptions.

Reply:
You are correct, the description could be better here. Let me think on this.